webroot/src/components/Licensee/LicenseeList/LicenseeList.vue

```diff
@@ -10,15 +10,29 @@
         <h1 class="list-title">{{ $t('licensing.licensingListTitle') }}</h1>
         <transition name="fade-slow" mode="out-in">
             <div v-if="!hasSearched" class="search-initial-container">
-                <LicenseeSearch @searchParams="handleSearch" />
+                <LicenseeSearch :searchParams="searchParams" @searchParams="handleSearch" />
             </div>
             <div v-else-if="shouldShowSearchModal" class="search-modal-container">
-                <LicenseeSearch @searchParams="handleSearch" />
+                <LicenseeSearch :searchParams="searchParams" @searchParams="handleSearch" />
             </div>
             <div v-else class="licesee-list-container">
                 <div class="search-toggle-container">
+                    <div v-if="hasSearchTerms" class="search-tag">
+                        <span class="title">{{ $t('common.viewing') }}:</span>
+                        <span class="search-terms">
+                            {{ searchDisplayFirstName }}
+                            {{ searchDisplayLastName }}
```

Comment:
I cant tell immediately why but there are spaces being added before the commas between term

Reply:
It looks like that is a function of having all the terms listed here on separate lines, the white space is bleeding in.

I might make a searchText computed concatenating all these.